tests/lax_scipy_test.py

```diff
@@ -198,12 +198,6 @@ def testIssue980(self):
     self.assertAllClose(np.zeros((4,), dtype=np.float32),
                         lsp_special.expit(x))
 
-  @jax.numpy_rank_promotion('raise')
-  def testIssue3758(self):
-    x = np.array([1e5, 1e19, 1e10], dtype=np.float32)
-    q = np.array([1., 40., 30.], dtype=np.float32)
-    self.assertAllClose(np.array([1., 0., 0.], dtype=np.float32), lsp_special.zeta(x, q))
-
```

Comment:
Note: I removed this test because it fails for the native `zeta` implementation. As noted in the reporting issue (#3758), both `scipy.special.zeta` and `tf.math.zeta` also fail this test; with this in mind I think it's not too big an issue if JAX's implementation does as well.